docs/sql-ref-syntax-ddl-alter-view.md

```diff
@@ -19,4 +19,69 @@ license: |
   limitations under the License.
 ---
 
-**This page is under construction**
+### Description
+The `ALTER VIEW` statement changes various auxiliary properties of a view.
+
+
+#### Rename view
+Rename the existing view. If the view name already exists in the database, an exception is thrown. This operation does 
+support moving the views cross databases. 
+##### Syntax
+{% highlight sql %}
+ALTER VIEW [db_name.]view_name RENAME TO [db_name.]new_view_name
+{% endhighlight %}
+
+
+#### Set view properties
+Set one or more properties of an existing view. The properties are the key value pairs. If the properties' keys exist, 
+the values are replaced with the new values. If the properties' keys does not exist, the key value pairs are added into 
+the properties.
+##### Syntax
+{% highlight sql %}
+ALTER VIEW view_name SET TBLPROPERTIES (key1=val1, key2=val2, ...)
+{% endhighlight %}
+
+
+#### Drop view properties
+Drop one or more properties of an existing view. If the specified keys do not exist, an exception is thrown. Use 
+`IF EXISTS` to avoid the exception. 
+##### Syntax
+{% highlight sql %}
+ALTER VIEW view_name UNSET TBLPROPERTIES [IF EXISTS] (key1=val1, key2=val2, ...)
+{% endhighlight %}
+
+
+#### Alter View As Select
+`ALTER VIEW AS SELECT` statement changes the definition of a view, the `select_statement` must valid, and the `VIEW` 
+must exist.
+##### Syntax
+{% highlight sql %}
+ALTER VIEW view_name AS select_statement
+{% endhighlight %}
+
+
+#### Example
+{% highlight sql %}
+-- Rename only change the view name.
+ALTER VIEW tempdb.view1 RENAME TO tempdb.view2
```

Comment:
If the source and target database has to be same, then should we just have the unqualified name in here ?

Reply:
yah, we can use qualified or unqualified name for the target view, I will add a note there. thanks.